src/Stratis.Features.FederatedPeg/OpReturnDataReader.cs

```diff
@@ -105,7 +107,7 @@ private string TryConvertValidOpReturnDataToAddress(byte[] data)
             }
             catch (Exception ex)
             {
-                this.logger.LogTrace($"Address {destination} could not be converted to a valid address. Reason {ex.Message}.");
+                this.logger.LogDebug($"Address {destination} could not be converted to a valid address. Reason {ex.Message}.");
```

Comment:
I think there is a rule about logging, saying that we shouldn't use string interpolation on logs

Reply:
see https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/using-logging.md#interpolated-strings

Reply:
Updated it to use structured logging

Reply:
yes no string interpolation as it evaluates the result even if logs are switched off.